libr/core/config.c

```diff
@@ -1028,6 +1028,20 @@ static int cb_rawstr(void *user, void *data) {
 	return true;
 }
 
+static int cb_binmaxstrrange(void *user, void *data) {
+	RCore *core = (RCore *) user;
+	RConfigNode *node = (RConfigNode *) data;
+	if (core->bin) {
+		int v = node->i_value;
+		if (v<1) v = 4; // HACK
+		core->bin->maxstrrange = v;
+	// TODO: Do not refresh if nothing changed
```

Comment:
What if the new value is smaller than the old one? Probably we don't need to refresh strings even in that case, because we already have more strings than requested and IMO it's ok to have more strings.
What do you think?